react/Field/index.jsx

```diff
@@ -245,6 +250,7 @@ Field.propTypes = {
 
 Field.defaultProps = {
   fieldProps: {},
+  labelProps: {},
```

Comment:
Nit: No need for default object here since labelProps is not required (spreading `...undefined` will not result in an error)